xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -695,6 +695,9 @@ bool CVideoPlayer::OpenFile(const CFileItem& file, const CPlayerOptions &options
 {
   CLog::Log(LOGNOTICE, "VideoPlayer::OpenFile: %s", CURL::GetRedacted(file.GetPath()).c_str());
 
+  if (m_omxplayer_mode && IsRunning())
```

Comment:
We have to look for a better way. I want to get rid of these omxplayer hacks instead of makeing it worse.

Reply:
What do you suggest?

Reply:
My knowledge about oxmplayer is limited. I am just looking at the facts:
- Currently VideoPlayer has 22 exceptions for omxplayer mode + some ifdefs for TARGET_RASPBERRY_PI
- This is only for a single platform: Raspberry Pi
- omxplayer mode is disabled by default for Pi versions > 1

Maybe we should just drop this mode? I don't see any good reason why anybody wants to run Kodi on a Pi 1. If there are any, they can stick with v17